mockwebserver/src/main/kotlin/mockwebserver3/MockWebServer.kt

```diff
@@ -828,7 +822,7 @@ public class MockWebServer : Closeable {
 
     val connectionClose = CountDownLatch(1)
     val streams =
-      object : RealWebSocket.Streams(false, source, sink) {
+      object : RealWebSocket.Streams(false, socket.source, socket.sink) {
```

Comment:
I assume eventually you'll pass socket down?

Reply:
EXACTLY!